hw/fpga/constraints/pynq-z2/pin_assign.xdc

```diff
@@ -2,83 +2,100 @@
 # Solderpad Hardware License, Version 2.1, see LICENSE.md for details.
 # SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
 
+# Clock signal
 set_property -dict {PACKAGE_PIN H16 IOSTANDARD LVCMOS33} [get_ports clk_i]
+create_clock -add -name sys_clk_pin -period 10.00 -waveform {0 5} [get_ports {clk_i}];
```

Comment:
this should go into the constraint file in the same folder

Reply:
but for the nexys we have it in the pin_assign file, should we change both to the respective constraint file?

Reply:
correct, then the nexys is also wrong sorry